python/mujoco/rollout.cc

```diff
@@ -37,7 +37,7 @@ const auto rollout_doc = R"(
 Roll out open-loop trajectories from initial states, get resulting states and sensor values.
 
   input arguments (required):
-    model              instance of MjModel
+    model              list of MjModel instances of length nroll
```

Comment:
- list -> array
- MjModel -> mjModel

Also, I think this function needs another argument that tells it if there is one model or `nroll` models, see my reasoning below. This should probably be `int nmodel` which is guaranteed to be either `nroll` or 1.

Reply:
You're right, I misread.